LastHW)/Write_article.py

```diff
@@ -0,0 +1,246 @@
+import os
+import re
+from collections import Counter
+from configparser import ConfigParser
+
+import datefinder
+import datetime
+import random
+
+import json
+import csv
+
+import xml.etree.ElementTree as ET
+import sqlite3
+import HW_3_updated as normalization
+
+
+# sys.path.append('C:\\Users\\Kateryna_Kulmatytska\\PycharmProjects\\Python_HW\\HW_6')
+
+class Common:
+    def __init__(self):
+        return
+
+    def get_header(self):
+        input_versions = ['News', 'PrivatAdv', 'Consider']
+        while True:
+            header = input(f"Enter the topic you are interested in: {', '.join(input_versions)}\n")
+            if header not in input_versions:
+                print('Please select correct topic')
+            else:
+                break
+        return header
+
+    def get_text(self):
+        print("Type your text here. Press Enter twice to exit from text adding mode")
+        input_text = ''
+        while True:
+            new_line = input()
+            if new_line != '':
+                input_text += new_line + '\n'
+            else:
+                input_text = input_text[:-1]
+                break
+        text = normalization.normalization(input_text)
+        return text
+
+    def get_attrib(self, header):
+        if header == 'News':
+            location = input('Enter location \n:')
+            return location
+        elif header == 'PrivatAdv':
+            try:
+                expiration_date = list(datefinder.find_dates(
+                    input('Till what date we should keep it?\nPlease follow DD-MM-YYYY date version:')))[0].date()
+            except:
+                print('Incorrect date format')
+                expiration_date = list(datefinder.find_dates(
+                    input('Till what date we should keep it?\nPlease follow DD-MM-YYYY date version:')))[0].date()
```

Comment:
1. The program will crash after 2nd incorrect date input.
2. According to documentation, **find_dates** returns a generator, so it'd be better to:
```
expiration_date = next(datefinder.find_dates(
                    input('Till what date we should keep it?\nPlease follow DD-MM-YYYY date version:'))).date()
```